cypress/support/pages/common_helper/globalAssertionSteps.js

```diff
@@ -16,6 +16,10 @@ class GlobalAssertionSteps {
 			cy.get(GlobalAssertionSteps.#firstElementOfBreadcrumb).parent().contains('li', content);
 		});
 	}
+
+	checkSidebar(activePage) {
+		cy.get(`[data-testid="sidebar-${activePage}"]`).should('have.class', 'v-list-item--active');
```

Comment:
this will not work for legacy pages

Reply:
Yes, currently it is only used for vue pages.
I'd say we extend it then, when we also need it for legacy, ok?

Reply:
its fine. one point to clarify in nuxt pages sidebar menu which is highlighted have this attribute in DOM: `aria-current="page"` but this attribute is missing when I navigate to folders details page.